### PR TITLE
refactor: Refactored the templates and github actions

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        4. See error
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: false
+  - type: textarea
```
Comment (patchandpray) [inline .github/ISSUE_TEMPLATE/bug_report.yml:29]: How do we expect a video or image to be added in a text area?

File: .github/ISSUE_TEMPLATE/doc_report.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: Documentation request
+description: Change regarding improvising the docs to be more accessible
+title: '[DOCUMENTATION]'
+labels: ['documentation']
+body:
+  - type: checkboxes
+    attributes:
+      label: Describe the bug
```
Comment (patchandpray) [inline .github/ISSUE_TEMPLATE/doc_report.yml:8]: Is the label accuracte here? Do we request somebody to describe a bug or do something else?

File: .github/ISSUE_TEMPLATE/doc_report.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      description: A clear and concise description of what the bug is
```
Comment (patchandpray) [inline .github/ISSUE_TEMPLATE/doc_report.yml:9]: Same here, this is a documentation request but we ask for a bug description?

File: .github/ISSUE_TEMPLATE/doc_report.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+name: Documentation request
+description: Change regarding improvising the docs to be more accessible
```
Comment (patchandpray) [inline .github/ISSUE_TEMPLATE/doc_report.yml:2]: improvising instead of 'improving' I think :)

File: .github/ISSUE_TEMPLATE/doc_report.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+title: '[DOCUMENTATION]'
+labels: ['documentation']
+body:
+  - type: checkboxes
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is
+      options:
```
Comment (patchandpray) [inline .github/ISSUE_TEMPLATE/doc_report.yml:10]: I'm not sure if options is a good idea here, what if a users option is not in here. Do we have an overview of commonly occuring options. I think we should keep it free form first and then if we start to see a pattern we can add checkboxes

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 6]
```diff
@@ -0,0 +1,24 @@
+name: Feature request
+description: Suggest an idea for this project
+title: '[FEATURE]'
+labels: ['enhancement']
+body:
+  - type: textarea
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
```
Comment (patchandpray) [inline .github/ISSUE_TEMPLATE/feature_request.yml:8]: I think we can remove the related to a problem bit. Just ask for a feature request. Let the requestor fill in if it is related to a problem.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+      description: A clear and concise description of what the problem is. Ex. Whenever I try to do (xyz) then [...]
```
Comment (patchandpray) [inline .github/ISSUE_TEMPLATE/feature_request.yml:9]: again, we have a feature request here but we ask a user to describe the problem.

I can agree that ususally a feature solves an issue or problem. but this feels a bit too bug focused versus innovation / improvement focused

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (kaiwalyakoparkar) [reply .github/ISSUE_TEMPLATE/bug_report.yml:29]: Actually every text area has a little option to upload an image or video 
Example image

File: .github/ISSUE_TEMPLATE/doc_report.yml (hunk 3 above)
Comment (kaiwalyakoparkar) [reply .github/ISSUE_TEMPLATE/doc_report.yml:9]: Sorry for that I will change it

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 6 above)
Comment (kaiwalyakoparkar) [reply .github/ISSUE_TEMPLATE/feature_request.yml:8]: Sure!

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (patchandpray) [reply .github/ISSUE_TEMPLATE/bug_report.yml:29]: Ok great!

File: .github/ISSUE_TEMPLATE/doc_report.yml (hunk 3 above)
Comment (patchandpray) [reply .github/ISSUE_TEMPLATE/doc_report.yml:9]: no reason to be sorry :)

File: .github/ISSUE_TEMPLATE/doc_report.yml (hunk 5 above)
Comment (patchandpray) [reply .github/ISSUE_TEMPLATE/doc_report.yml:10]: Hey @kaiwalyakoparkar you resolved this conversation but I the options are still there. What is your reasoning behind this? :)

File: .github/ISSUE_TEMPLATE/doc_report.yml (hunk 4 above)
Comment (patchandpray) [reply .github/ISSUE_TEMPLATE/doc_report.yml:2]: still improvising versus improving :D

Comment (kaiwalyakoparkar) [reply .github/ISSUE_TEMPLATE/doc_report.yml]: Done

File: .github/ISSUE_TEMPLATE/doc_report.yml (hunk 5 above)
Comment (kaiwalyakoparkar) [reply .github/ISSUE_TEMPLATE/doc_report.yml]: Oops, so sorry for that. I have changed it to simple text box now

**Review by patchandpray — APPROVED** [review]

**Pull request merged**